nbtools/pylint_notebook.py

```diff
@@ -117,25 +121,40 @@ def pylint_notebook(path=None, options=(), disable=(), enable=(), printer=print,
     pylintrc = generate_pylintrc(path_pylintrc, disable=disable, enable=enable, **pylint_params)
 
     # Run pylint on script with pylintrc configuration
-    pylint_cmdline = ' '.join([path_pylintrc, f'--rcfile {path_pylintrc}', *options])
-    pylint_stdout, pylint_stderr = lint.py_run(pylint_cmdline, return_std=True)
+    pylint_cmdline = [path_pylintrc, f'--rcfile={path_pylintrc}', *options]
+
+    # Custom streams for catching messagies
+    pylint_stdout = StringIO()
+
+    base_stderr = sys.stderr
```

Comment:
Can we use `from contextlib import redirect_stdout`?

Reply:
Thanks, that much better